Budget/budgetRouter.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+
+const Budget = require('./Budget.js');
+
+const router = express.Router();
+
+router.post('/', (req, res) => {
+    const {title, budgetAmount} = req.body
```

Comment:
Since we're not using the variables `title` and `budgetAmount` for anything, we could just pass `req.body` to our new Budget: `const budget = new Budget(req.body)` and skip the whole object destructuring thing.